pkgs/applications/audio/argotlunar/default.nix

```
@@ -0,0 +1,74 @@
+ { stdenv
+ , lib
+ , fetchFromGitHub
+ , alsa-lib
+ , freetype
+ , curl
+ , pkg-config
+ , cmake
+ , xorg
+ }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "argotlunar";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "mourednik";
+    repo = "argotlunar";
+    rev = "1527d5dd753a7af32600ac677337469bbf1d40cd";
```

Why not parameterize it, like

```suggestion
    rev = "V${version}";
```

?